packages/react-core/src/components/Modal/ModalBox.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { css } from '@patternfly/react-styles';
+import PropTypes from 'prop-types';
+import styles from '@patternfly/patternfly-next/components/ModalBox/styles.css';
+
+const propTypes = {
+  /** content rendered inside the button. */
+  children: PropTypes.node.isRequired,
+  /** additional classes added to the button */
+  className: PropTypes.string,
+  /** Creates a large version of the ModalBox */
+  isLarge: PropTypes.bool,
+  /* Aria label used for Modal Box Header */
+  label: PropTypes.string.isRequired
+};
+
+const defaultProps = {
+  className: '',
+  isLarge: false
+};
+
+const ModalBox = ({ children, className, isLarge, label, ...props }) => (
+  <div
+    {...props}
+    role="dialog"
+    aria-label={label}
```

Comment:
I think we are going to need to switch between aria-label and aria-labeledby based on if there is a title for the modal or not. @jgiardino  is this correct?

Reply:
Yes, I made a similar comment about this above.